coral/config/genesis.json

```diff
@@ -52,6 +52,25 @@
             "account_number": "0",
             "sequence": "0"
           }
+        },
+        {
+          "type": "cosmos-sdk/Account",
+          "value": {
+          "address": "cosmos1xgf0cm289z0z2w5phhy6ay3xscq5fpmt8xefhn",
+          "coins": [
+            {
+              "denom": "ureef",
+              "amount": "100000000"
+            },
+            {
+              "denom": "ushell",
+              "amount": "100000000"
+            }
+          ],
+          "public_key": null,
+          "account_number": "0",
```

Comment:
This does not look right. We already have 0 above. But I am not an expert in genesis files

Reply:
Account numbers start at 1. If I understand properly, new accounts always have 0 which mean "assign new auto-increment number". If you do a genesis export/import, you will have plenty of non-zero account_number and sequence number.

This is just how cosmos works, not a bug.

Reply:
yeah,  I am using command `coral add-genesis-account` to add my address, the `account_number` always be "0"